### DISCUSSION TITLE
cueobserve repo

### DISCUSSION THREAD

**Discussion by jithendra945** [cuebook/CueObserve#97] [Q&A] [answered]
i pulled the changes, and saw them on vscode.

i tried pull on docker, as it is having latest tag, I deleted the container and run the command again, still the code is not updated when i check on docker container > ops (folder) > code dint updated.

in vscode, ops > tasks tests init.py
in docker container > anomalyDetection.py  tasks.py  tests

Any solution ?

**Comment by vincue** [comment, marked as answer]
The latest tag is not deployed continuously. You can build docker image yourself as mentioned in [documentation](https://cueobserve.cuebook.ai/development#building-docker-image).
